<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transaksi_pembayaran</class>
 <widget class="QDialog" name="transaksi_pembayaran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>766</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>766</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #6a11cb, stop:1 #2575fc);
</string>
  </property>
  <widget class="QFrame" name="kereta_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>1041</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 150); /* Transparansi */
border-radius: 15px; /* Membuat sudut melengkung */
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="kursi_kereta_box" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>971</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(158, 213, 255);</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>921</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width: 25px;  /* Lebar kotak */
    height: 25px; /* Tinggi kotak */
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_saldo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>801</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background : white;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_saldouser">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background : white;</string>
       </property>
       <property name="text">
        <string>SALDO ANDA </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_saldo">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>0</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background : white;</string>
       </property>
       <property name="text">
        <string>SALDO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_inputrekening">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>801</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background : white;</string>
      </property>
      <property name="text">
       <string>          MASUKAN PIN REKENING ANDA </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_harga">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background : white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_pinNorek">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>120</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    border: none;
    border-bottom: 2px solid black;
	border-radius: 0px;
    color: black;
    font-size: 16px;
    padding: 5px;
    background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>masukkan PIN anda .... </string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TRANSAKSI PEMBAYARAN </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_data_">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>971</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(158, 213, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;PESAN TIKET &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_keluar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>680</y>
      <width>971</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(44, 181, 255);
    color: white;
    border-radius: 10px;
	font-family: 'Montserrat';
    font-weight: bold;
    font-size: 14pt; 
    padding: 8px;
}
QPushButton:hover {
    background-color: #0056b3;
}
</string>
    </property>
    <property name="text">
     <string>KELUAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_bayar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>971</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(44, 181, 255);
    color: white;
    border-radius: 10px;
	font-family: 'Montserrat';
    font-weight: bold;
    font-size: 14pt; 
    padding: 8px;
}
QPushButton:hover {
    background-color: #0056b3;
}
</string>
    </property>
    <property name="text">
     <string>BAYAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_waktu_ticket">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TANGGAL KEBERANGKATAN DAN WAKTU KERETA </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_nama_kereta">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NAMA KERETA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_layanan">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>JENIS LAYANAN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_penumpang">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1 PENUMPANG </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_container_asaltujan">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>771</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_asal">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ASAL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_tujuan">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TUJUAN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_harga_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HARGA KERETA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_kursi">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID KURSI</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>→ </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>766</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>766</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>kereta_box</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
